harness/tests/experiment/pytorch/test_deepspeed_autotuning.py

```diff
@@ -242,7 +235,7 @@ def _append_events_for_op(self, op: searcher.Operation) -> None:
             self.events_queue.append(event)
 
         elif isinstance(op, searcher.Shutdown):
-            exp_state = bindings.experimentv1State.STATE_COMPLETED
+            exp_state = bindings.experimentv1State.COMPLETED
```

Comment:
what's up with the various enum changes? and isn't `STATE_COMPLETED` the right value here? I'm seeing
```python
class experimentv1State(enum.Enum):
    STATE_UNSPECIFIED = "STATE_UNSPECIFIED"
    STATE_ACTIVE = "STATE_ACTIVE"
    STATE_PAUSED = "STATE_PAUSED"
    STATE_STOPPING_COMPLETED = "STATE_STOPPING_COMPLETED"
    STATE_STOPPING_CANCELED = "STATE_STOPPING_CANCELED"
    STATE_STOPPING_ERROR = "STATE_STOPPING_ERROR"
    STATE_COMPLETED = "STATE_COMPLETED"
    STATE_CANCELED = "STATE_CANCELED"
    STATE_ERROR = "STATE_ERROR"
    STATE_DELETED = "STATE_DELETED"
    STATE_DELETING = "STATE_DELETING"
    STATE_DELETE_FAILED = "STATE_DELETE_FAILED"
    STATE_STOPPING_KILLED = "STATE_STOPPING_KILLED"
    STATE_QUEUED = "STATE_QUEUED"
    STATE_PULLING = "STATE_PULLING"
    STATE_STARTING = "STATE_STARTING"
    STATE_RUNNING = "STATE_RUNNING"
```

Reply:
They changed it here: https://github.com/determined-ai/determined/pull/6496 The bindings are the same but some of the names to python internal APIs are shortened now